server/datastore/mysql/migrations/tables/migration.go

```diff
@@ -27,6 +27,22 @@ AND CONSTRAINT_NAME = ?
 	return count > 0
 }
 
+func unqConstraintExists(tx *sql.Tx, table string, name string) bool {
+	var count int
+	err := tx.QueryRow(`
+SELECT COUNT(1)
+FROM information_schema.TABLE_CONSTRAINTS
+WHERE TABLE_NAME = ? AND CONSTRAINT_TYPE = 'UNIQUE' AND CONSTRAINT_NAME = ?
+`,
+		table, name,
+	).Scan(&count)
+	if err != nil {
+		return false
```

Comment:
Should we check for `sql.ErrNoRows` and have this method return `(error, bool)`. Just in case, to not masquerade an error when checking information_schema tables as "constraint does not exist".

Reply:
Done in https://github.com/fleetdm/fleet/pull/10514/commits/0954c57d230eb528bd55907d67fc12cc87699085